src/systems/eu/eu-component/eu-component-global-navigation/eu-component-global-navigation.scss

```diff
@@ -0,0 +1,177 @@
+  font-size: map-get($ecl-font-size, 's');
+  margin: 0;
+}
+
+.ecl-global-navigation__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.ecl-global-navigation__option {
+  flex-grow: 1;
+}
+
+.ecl-global-navigation__link {
+  color: map-get($ecl-colors, 'grey-100');;
+  display: block;
+  line-height: 1.3;
+  padding: map-get($ecl-spacing, 'xxs') map-get($ecl-spacing, 's');
+  text-decoration: none;
+  width: 100%;
+
+  &:hover,
+  &:focus {
+    text-decoration: underline;
+  }
+
+  &:visited {
+    color: map-get($ecl-colors, 'grey-100');;
+  }
+}
+
+.ecl-global-navigation__link--official {
+  align-items: center;
+  display: flex;
+  position: relative;
+
+  .ecl-global-navigation__label {
+    padding-left: map-get($ecl-spacing, 'xl');
+  }
+
+  &::before {
+    background: url($ecl-assets-path+'images/logo.svg') no-repeat;
+    background-size: contain;
+    content: '';
+    display: block;
+    height: 25px;
+    position: absolute;
+    top: 50%;
+    transform: translateY(-50%);
+    width: 37px;
+  }
+}
+
+.ecl-global-navigation__link--institution {
+  background-color: map-get($ecl-colors, 'grey-25');
+}
+
+.ecl-global-navigation__toggle {
+  background-color: map-get($ecl-colors, 'grey-25');
+  padding: map-get($ecl-spacing, 's') map-get($ecl-spacing, 'm');
+}
+
+.ecl-global-navigation__toggle-content {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.ecl-global-navigation__institution-list {
+  display: flex;
+  flex-wrap: wrap;
+  list-style: none;
+  margin: 0;
+  order: 1;
+  padding: 0;
+}
+
+.ecl-global-navigation__institution-item {
+  width: 100%;
+}
+
+.ecl-global-navigation__institution-link {
+  color: map-get($ecl-colors, 'grey-100');
+  display: block;
+  padding: map-get($ecl-spacing, 'xxs') 0;
+
+  &:visited {
+    color: map-get($ecl-colors, 'grey-100');
+  }
+
+  &:hover,
+  &:focus {
+    color: #000;
+  }
+}
+
+.ecl-global-navigation__info {
+  color: map-get($ecl-colors, 'grey-100');
+  flex-grow: 1;
+  font-weight: 600;
+  order: 0;
+  padding: map-get($ecl-spacing, 'xxs') 0;
+}
+
+/* Dark display */
+.ecl-global-navigation--dark {
+  background-color: map-get($ecl-colors, 'grey-100');
+
+  .ecl-global-navigation__link {
+    color: #fff;
+  }
+
+  .ecl-global-navigation__link:visited {
+    color: #fff;
+  }
+
+  .ecl-global-navigation__link--institution {
+    background-color: map-get($ecl-colors, 'grey-75');
+  }
+
+  .ecl-global-navigation__toggle {
+    background-color: map-get($ecl-colors, 'grey-75');
+  }
+
+  .ecl-global-navigation__institution-link {
+    color: #fff;
+  }
+
+  .ecl-global-navigation__institution-link:visited {
+    color: #fff;
+  }
+
+  .ecl-global-navigation__institution-link:hover,
+  .ecl-global-navigation__institution-link:focus {
+    color: map-get($ecl-colors, 'grey-5');
+  }
+
+  .ecl-global-navigation__info {
+    color: #fff;
+  }
+}
+
+@include ecl-media-breakpoint-up(md) {
+  .ecl-global-navigation__institution-item {
+    width: 50%;
```

Comment:
It could be good to apply a horizontal padding on list items (something between 0.5rem and 1rem) and apply the same but negatively margin on the parent:

```
.ecl-global-navigation__institution-list { margin: 0 -0.5rem; }
.ecl-global-navigation__institution-item { padding: 0 0.5rem; }
```

It would avoid this kind of situation:

![no-gutter](https://user-images.githubusercontent.com/3524389/37766421-2db5b146-2dc7-11e8-8dc1-a1ed2872442e.jpg)